src/k2/cmd/ycsb/data.h

```diff
@@ -178,7 +178,7 @@ class YCSBData{
 
     static inline seastar::future<> waitSchemasLoaded(K23SIClient& client) {
         return seastar::do_with(
-            Deadline(20s),
+            Deadline(400s),
```

Comment:
ouch, is that really needed?

Reply:
I was getting timeouts on the collection creation and schema load, I bumped them up across the board (most were config variables). I don't know exactly how long we need. I figure we can look into it more deeply when we think about a CPO rework.